From db69bf6ecd0da4d23bdfe38652bb53d2daa655a2 Mon Sep 17 00:00:00 2001
From: Akira TAGOH <akira@tagoh.org>
Date: Fri, 1 Mar 2013 18:31:01 +0900
Subject: [PATCH 2/2] Bug 60783 - Add Liberation Sans Narrow to
 30-metric-aliases.conf

Add Liberation Sans Narrow as an alias for Arial Narrow
---
 conf.d/30-metric-aliases.conf | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/conf.d/30-metric-aliases.conf b/conf.d/30-metric-aliases.conf
index ccc3e24..0d118b2 100644
--- a/conf.d/30-metric-aliases.conf
+++ b/conf.d/30-metric-aliases.conf
@@ -15,10 +15,12 @@
 
 		Microsoft fonts:
 			Arial
+			Arial Narrow
 			Times New Roman
 			Courier New
 		Liberation fonts:
 			Liberation Sans
+			Liberation Ssns Narrow
 			Liberation Serif
 			Liberation Mono
 		StarOffice fonts:
@@ -93,6 +95,13 @@
 	</alias>
 
 	<alias binding="same">
+	  <family>Liberation Sans Narrow</family>
+	  <default>
+	    <family>Arial Narrow</family>
+	  </default>
+	</alias>
+
+	<alias binding="same">
 	  <family>Albany</family>
 	  <default>
 	  <family>Arial</family>
@@ -252,6 +261,13 @@
 	</alias>
 
 	<alias binding="same">
+	  <family>Arial Narrow</family>
+	  <accept>
+	    <family>Liberation Sans Narrow</family>
+	  </accept>
+	</alias>
+
+	<alias binding="same">
 	  <family>Times New Roman</family>
 	  <accept>
 	    <family>Tinos</family>
-- 
1.8.1.5

